apps/docs/pages/guides/auth/auth-user-management.mdx

```diff
@@ -0,0 +1,130 @@
+import Layout from '~/layouts/DefaultGuideLayout'
+
+export const meta = {
+  id: 'auth-user-management',
+  title: 'User Management',
+  description: 'Manage your users with Supabase Auth',
+  subtitle: 'Manage your users with Supabase Auth',
+}
+
+## The User Object
+The user object stores all the information related to a user in your application. The user object can be retrieved using one of these methods:
+1. Retrieve the logged in user object using [`supabase.auth.getUser()`](/docs/reference/javascript/auth-getuser) 
```

Comment:
/nit consider dropping  the prefix: `Retrieve the logged in user object using` since the preamble specifies that the options are methods
```suggestion
1. [`supabase.auth.getUser()`](/docs/reference/javascript/auth-getuser) 
```